### PR TITLE
Use incident ID to keep track of rendered maps

### LINKED ISSUE #38: Map for multiple victim results not rendered
The `MapDisplay` component uses a div with hardcoded `#map` ID for providing rendering context to mapbox. As a result, if there are multiple maps on the same page, they do not get rendered. It could be solved using a unique `ID` for keeping track of each rendered map.

### PR DESCRIPTION
**Pull request by rottencandy**
- Use Incident ID instead of hardcoded ID for map divs
- Keep dict of all the rendered maps on the page
- Expose function which allows adding of additional markers to maps using their IDs

Fixes #38

### CODE REVIEW

File: src/components/DisplayList.tsx [hunk 1]
```diff
@@ -54,7 +54,7 @@ const DisplayList: React.FC<DisplayListProps> = ({ data }) => {
           <CardBody>
             <Split gutter="md">
               <SplitItem>
-                <MapDisplay lat={detail.lat} lon={detail.lon} />
+                <MapDisplay id={detail.id} lat={detail.lat} lon={detail.lon} />
```
Comment (invincibleJai) [inline src/components/DisplayList.tsx]: can destructre `id, lat, lon` and use

File: src/components/MapDisplay.tsx [hunk 2]
```diff
@@ -5,24 +5,34 @@ import './MapDisplay.css'
 
 const LIGHT_RED = chart_global_danger_Color_100.value
 
+const mapsList: { [id: string]: mapboxgl.Map } = {}
```
Comment (invincibleJai) [inline src/components/MapDisplay.tsx]: can we have type for `{ [id: string]: mapboxgl.Map }`

File: src/components/DisplayList.tsx (hunk 1 above)
Comment (rottencandy) [reply src/components/DisplayList.tsx]: Updated.

File: src/components/MapDisplay.tsx (hunk 2 above)
Comment (rottencandy) [reply src/components/MapDisplay.tsx]: Added.

**Pull request merged**